CS342MemberViews/Sources/BrookeKrajancich/ProfilePanel.swift

```diff
@@ -0,0 +1,73 @@
+// swift-tools-version: 5.7
+
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+struct ProfilePanel: View {
+    private var profileImage: Image {
+        guard let imagePath = Bundle.module.path(forResource: "BrookeKrajancich", ofType: "png"),
+           let image = UIImage(contentsOfFile: imagePath) else {
+            return Image(systemName: "person.fill")
+        }
+        
+        return Image(uiImage: image)
+    }
+    
+    private var backgroundImage: Image {
+        guard let imagePath = Bundle.module.path(forResource: "Background", ofType: "jpg"),
+           let image = UIImage(contentsOfFile: imagePath) else {
+            return Image(systemName: "person.fill")
+        }
+        
+        return Image(uiImage: image)
+    }
+    
+    var backgroundColor = Color(red: 0.067, green: 0.094, blue: 0.153)
+//    var highlightColor2: Color = Color(hue: 0.827, saturation: 0.669, brightness: 0.908)
+    var highlightColor = Color(red: 0.96, green: 0.45, blue: 0.71)
```

Comment:
You can make this property a constant as it is never modified:
```suggestion
    let highlightColor = Color(red: 0.96, green: 0.45, blue: 0.71)
```